sources/meso.json

```diff
@@ -0,0 +1,10 @@
+{
+	"id": 1108756907,
+	"fullname": "Mesoshapes",
+	"name": "meso",
+	"prefix": "meso",
+	"key":"",
+	"url": "https://github.com/whosonfirst-data/whosonfirst-data/blob/master/LICENSE.md",
```

Comment:
This document doesn't reference Mesoshapes at all, but should. So link here is okay, but add new issue in that repo to add Mesoshapes to that LICENSE file.